mLRS/Common/fhss.cpp

```diff
@@ -71,7 +71,7 @@ void FhssBase::generate(uint32_t seed)
         // ensure it is not too close to the previous
         // do only if we have plenty of channels at our disposal
         bool is_too_close = false;
-        if ((config_i != FHSS_CONFIG_433_MHZ) && (k > 0)) { // TODO: use smarter method, e.g., cnt < 2/3
+        if (((config_i != FHSS_CONFIG_433_MHZ) || (config_i !=FHSS_CONFIG_866_MHZ_IN)) && (k > 0)) { // TODO: use smarter method, e.g., cnt < 2/3
```

Comment:
shouldn't this be && ??

Reply:
I thought there would only be 1 frequency value stored in config_i so we'd have to check for 433 _OR_ 866  IN? I could be wrong :)